stable/ipfs/templates/statefulset.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apps/v1beta1
+kind: StatefulSet
+metadata:
+  name: {{ template "name" . }}
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+spec:
+  replicas: {{ .Values.replicaCount }}
+  serviceName: {{ template "name" . }}
+  volumeClaimTemplates:
+    - metadata:
+        name: ipfs-storage
+      spec:
+        accessModes: [ "ReadWriteOnce" ]
+        resources:
+          requests:
+            storage: {{ default "10Gi" .Values.storage.size }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "fullname" . }}
+    spec:
+      securityContext:
+        fsGroup: 1000
+        runAsUser: 1000
+      containers:
+      - name: {{ .Chart.Name }}
+        image: jbenet/go-ipfs:release 
```

Comment:
Probably want to use `ipfs/go-ipfs` instead of from Juan's account, and also lock it to a version, `v0.4.9` is latest available in Docker Hub.